Add lazer mod articles (#12105)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Muted/en.md

---
stub: true
tags:
  - MU
---

# Muted (lazer mod)

::: Infobox

<!-- lint ignore heading-increment -->

#### Muted

> ![No Scope mod icon](/wiki/Gameplay/Game_modifier_(lazer)/img/mods/MU.png)

Reviewer: ```suggestion
![Muted mod icon](/wiki/Gameplay/Game_modifier_(lazer)/img/mods/MU.png)
```

*Can you still feel the rhythm without music?*

|  |  |
| :-- | :-- |
| Acronym | MU |
| Type | Fun |

> | Game modes | ![][osu!] ![osu!taiko] ![][osu!catch] ![osu!mania] |

Reviewer: ```suggestion
| Game modes | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
```

| Score multiplier | 1.00x |
| Status | Ranked |
| Incompatible mods | None |

:::

*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

## Description

> The **Muted** mod progressively removes the audio between the first and hundredth combo. Beyond that number, only the metronome can be heard for the rest of the map. If you miss an object, everything will be back to normal.

Reviewer: ```suggestion
The **Muted** mod progressively removes the audio while the combo ranges between 1 and a set value. Beyond that range, only the metronome can be heard for the rest of the beatmap. If a [hit object](/wiki/Gameplay/Hit_object) is missed, the audio will be back to normal.
```

## Customise

![Customise Muted](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/MU.jpg)

> - **Start muted:** The audio will be muted when you start the map. The more combo you get, the better you can hear the track. *The option is disabled by default.*

Reviewer: Would it make sense to create a table for these settings?

Like so maybe:

| Mod setting | Description | Default |
| :-: | :-- | :-: |
| Test | This is a test. | 0 |

Reviewer 2: paragraphs inside tables are pretty bad UX for some clients (mobile) and we have no way to fix that yet

> - **Start muted:** The audio will be muted when you start the map. The more combo you get, the better you can hear the track. *The option is disabled by default.*

Reviewer: ```suggestion
- **Start muted:** The mod's effect is flipped around by increasing the audio's volume the more combo a player gets. *The option is disabled by default.*
```

- **Enable metronome:** A metronome will be added to help you keep on track of the rhythm. *The option is enabled by default.*

> - **Finale volume at combo:** You can change the value of the combo in which the audio will be removed. It ranges from 0 (always muted) to 500. *100 is the default value.*

Reviewer: ```suggestion
- **Finale volume at combo:** The combo range in which the audio will be removed. It ranges from 0 (always muted) to 500. *100 is the default value.*
```

> - **Mute hit sound:s** When the volume gradually decreases the more combo you get. the hit sounds will also be muted. *The option is enabled by default.*

Reviewer: ```suggestion
- **Mute hit sounds:** Decrease the volume of hit sounds together with the music. *The option is enabled by default.*
```
